enstore-storage/enstore-storage

```diff
@@ -253,6 +253,8 @@ date cannot be null.
         #end_time = timeutil.at_minute(datetime.datetime.now())
         end_time = None
         # Give to the end time a 60 seconds delay. To allow for enstore DB processing time
+        if limit is None:
+            limit = 0
         if limit > 0:
```

Comment:
```suggestion
        elif limit > 0:
```

No reason to check this if it's originally `None` and we explicitly set it to 0

Reply:
That's true but changing this to an `elif` like you have it would actually break the logic he was trying to fix, as it would prevent the `else` block a few lines down from firing when `limit` _is_ `None`.

Reply:
PS i think this spot must have been the python2/3 issue Yujun had in mind, as `None > 0` worked as a comparison in python2 but raises an exception in python3 (which is why he added this explicit None handling).